src/content/1.7/testing/web-acceptance-tests/how-to-contribute-and-create-web-acceptance-tests.md

```diff
@@ -0,0 +1,212 @@
+- Checkout page (which is part of the FO) extends from `FOBasePage`  
+
+#### Methods
+Methods inside a page class must be properly named about their primary function. Remember to [KISS](https://en.wikipedia.org/wiki/KISS_principle) ! You should be able to understand what a method does just by looking at its name.
+
+Methods inside a page class **CANNOT USE** the `expect` keyword or any kind of asserting function, since their sole purpose is to interact with the page they’re linked with. They don’t validate anything per se, they just expose functionalities. 
+To validate a behavior in your test logic, your methods can return booleans, integers (number of lines, elements, etc), strings, objects... It’s up to you.
+
+Examples: 
+```
+   /**
+   * Get order status
+   * @return {Promise<string>}
+   */
+  async getOrderStatus() {
+    return this.getTextContent(`${this.orderStatusesSelect} option[selected='selected']`, false);
+  }
+```
+
+#### Selectors
+Selectors are used on every page and are stored as attributes of the class. They should be named following this convention:
+
+**nameType*** in `camelCase`.
+
+For example, a button used to submit the main form in the order page should be named: `submitMainFormButton`.
+
+
+#### Types
+Each selector must belong to a certain type. Here is a non-exhaustive list:
+- button
+- link
+- block
+- image
+- icon
+- text
+- modal
+- *other HTML elements...*
+
+
+- table
+- table-header
+- table-cell
+- table-row
+- *other table elements...*
+
+
+- form
+- input
+- select
+- radio
+- checkbox
+- *other forms inputs...*
+
+### Tests
+
+#### Campaigns
+We currently have 2 campaigns implemented:
+
+- **Sanity**: its purpose is to validate a Pull Request. Executed on [Travis CI](https://travis-ci.com/), this campaign must fully pass before merging the PR (one failed test blocks the merge). It consists of a few tests of the core features of PrestaShop: shop installation, orders/products pages in BO, and catalog/cart/checkout process in FO.
+- **Functional**: it is the biggest and most important campaign. Its purpose is to validate that every feature of PrestaShop works, by testing them one by one. It goes on every page and tests whatever it can: table (filtering, ordering, quick edits, etc), [CRUD](https://en.wikipedia.org/wiki/Create,_read,_update_and_delete) items (orders, customers, products…), setting changes, etc.
+
+We plan on implementing 2 more campaigns:
+- **End to end**: its purpose is to check that popular user paths are working as intended. It will walk through the application and mimic a real user working on their daily routines as a merchant: checking products, generating invoices, creating a customer account, and an order, adding a special voucher for a specific user, etc. The selected user paths will be chosen by the Product Team. Thanks to their merchants and agencies interviews, they have a pretty good idea of what merchants do every day and how they use the software.
```

Comment:
Again, separate the first list item from the previous paragraph.
```suggestion

- **End to end**: its purpose is to check that popular user paths are working as intended. It will walk through the application and mimic a real user working on their daily routines as a merchant: checking products, generating invoices, creating a customer account, and an order, adding a special voucher for a specific user, etc. The selected user paths will be chosen by the Product Team. Thanks to their merchants and agencies interviews, they have a pretty good idea of what merchants do every day and how they use the software.
```